config/environments/development.rb

```diff
@@ -34,6 +34,24 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :test
+  host = 'example.com' # Don't use this literally; use your local dev host instead
```

Comment:
cái này môi trường development của mình thì phải thiết lập lại e nha
```suggestion
  host = "localhost:3000"
```